src/ducks/Studio/Alerts/suggestions/index.js

```diff
@@ -9,5 +14,26 @@ export const Suggestion = Object.assign(Object.create(null), {
   daily_active_addresses: {
     title: 'DAA',
     suggesters: dailyActiveAddressesSuggesters
+  },
+  social_volume_total: {
+    title: 'Social Volume',
+    suggesters: [
+      suggestValueChange(Metric.social_volume_total),
+      suggestPercentUpDown(Metric.social_volume_total)
```

Comment:
Since this is a generic pattern, these functions should be moved outside of the `suggestions/price` module to `suggestions/helpers`